deploy/helm/flightctl/templates/flightctl-db-persistentvolumeclaim.yaml

```diff
@@ -15,5 +15,5 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 2Gi
+      storage: 20Gi
```

Comment:
should this be configurable?

Reply:
Can we make the same Values.db.storage.size: 20Gi ?